packages/design-system/src/components/tabs.tsx

```diff
@@ -30,8 +30,8 @@ export const TabsTrigger = styled(TabsPrimitive.Trigger, {
   },
   [`& > ${Text}`]: {
     color: "$gray9",
-    fontSize: "11px",
-    fontWeight: "600",
+    fontSize: "calc($fontSizes$1 - 1px)",
```

Comment:
I wonder why we would need that -1px thing, it should probably just use one of the predefined fontSizes